Source/Generator/README.md

```diff
@@ -3,53 +3,40 @@
 This repo contains the LabVIEW Measurement template code
 
 ## Dependency List
+
 1. LabVIEW 2020 64 bit
-2. [LabVIEW gRPC Support](https://github.com/ni/grpc-labview/releases)
-This repo contains the support for LabVIEW gRPC. Download and install the two latest release packages for,
-2.1 ni_lib_labview_grpc_library-x.x.x.x.vip
-2.2 ni_lib_labview_grpc_servicer-x.x.x.x.vip
+2. [LabVIEW gRPC Support](https://github.com/ni/grpc-labview/releases) This repo contains the support for LabVIEW gRPC. Download and install the two latest release packages for
+    1. ni_lib_labview_grpc_library-x.x.x.x.vip
+    2. ni_lib_labview_grpc_servicer-x.x.x.x.vip
 
 ## Developer Guide
 
 ### Instantiate a New Measurement
 
 1. Create a new LabVIEW project (say `Measurement.lvproj`) and open it.
-2. Open the New Measurement Instantiation window, from the project window, goto `Tools` -> `TAF` -> `Measurement Plugin` -> `New...`
-2.1 In the New window, Enter the `Measurement Service Name` that you would like to give to the measurement and click `Create Measurement Service`.
-2.2 This will create a new measurement service library in the project.
+2. Open the New Measurement Instantiation window, from the project window, goto `Tools` -> `MeasurementLink` -> `Create Measurement Service...`
+    1. In the New window, Enter the `Measurement Service Name` that you would like to give to the measurement and click `Create Measurement Service`.
+    2. This will create a new measurement service library in the project.
 
 ### Run the Measurement Service
 
 To run the measurements, follow the below steps
-1. Open `<MeasurementName>.lvproj` that contains the Measurement Service library.
-3. Run the `Run Service.vi` from `<MeasurementName>.lvlib` to run the measurement service from `<MeasurementName>.lvproj`
+
+1. Open `<MeasurementName>.lvproj` that contains the measurement service library.
+2. Run the `Run Service.vi` from `<MeasurementName>.lvlib` to run the measurement service from `<MeasurementName>.lvproj`
 
 ### Stop the Measurement Service
 
-Measurement Service will be stopped automaticlly when the client frameworks calls the `ServiceManagementService\Stop Service` method on the Measurement Service. To manually trigger the stop, go to `Run Service.vi` of the respective measurements, and click on `Stop` button from the Front Panel.
+The measurement service will be stopped automatically when the client frameworks calls the `ServiceManagementService\Stop Service` method on the measurement service. To stop the service manually, click on the `STOP` button on the front panel.
```

Comment:
This is not true anymore, right? I thought we removed all of the service management stuff.

Reply:
Chopped that sentence.